maui/src/Expander/SfExpander.cs

```diff
@@ -1517,8 +1517,21 @@ protected override Size MeasureContent(double widthConstraint, double heightCons
 				}
 			}
 
+			double width = double.IsFinite(widthConstraint) ? widthConstraint : 0;
+			// To update width when loaded expander inside HorizontalStackLayout and AbsoluteLayout.
+			if (width == 0)
+			{
+#if !WINDOWS
+				var scaledScreenSize = new Size(DeviceDisplay.MainDisplayInfo.Width / DeviceDisplay.MainDisplayInfo.Density, DeviceDisplay.MainDisplayInfo.Height / DeviceDisplay.MainDisplayInfo.Density);
```

Comment:
nit: 

```suggestion
                var scaledScreenSize = 
#if WINDOWS
                new Size(300, 300);
#else
                new Size(DeviceDisplay.MainDisplayInfo.Width / DeviceDisplay.MainDisplayInfo.Density, DeviceDisplay.MainDisplayInfo.Height / DeviceDisplay.MainDisplayInfo.Density);
#endif
```

IMO it reads better and it would be easier to scale if we adapt it to more platforms in the future

Reply:
@bricefriha - Thanks for the suggestion — we have updated the code accordingly.